changelogs/fragments/5193-consul-session-token.yaml

```
@@ -0,0 +1,5 @@
+minor_changes:
+  - consul - adds ``token`` parameter for session  (https://github.com/ansible-collections/community.general/pull/5193).
```

This is only for the consul_session module, right?
```suggestion
  - consul_session - adds ``token`` parameter for session (https://github.com/ansible-collections/community.general/pull/5193).
```